src/libcore/ptr/mod.rs

```diff
@@ -647,42 +647,47 @@ pub unsafe fn read<T>(src: *const T) -> T {
 /// [read-ownership]: ./fn.read.html#ownership-of-the-returned-value
 /// [valid]: ../ptr/index.html#safety
 ///
-/// # Examples
```

Comment:
With this, now there is *no* example of valid usage of `read_unaligned`. That seems bad?

Reply:
But there was no valid example before either? That is, the only existing example was actively misleading. I would suggest that someone add an actual valid example in a follow up.

Reply:
That is fair. But at least it "looked" like there was one. ;)

Can you open an issue about adding a valid example, and Cc some of the usual suspects?

Reply:
Done, https://github.com/rust-lang/rust/issues/62357.